caffeine@patapon.info/extension.js

```diff
@@ -100,6 +100,12 @@ const DBusSessionManagerInhibitorProxy = Gio.DBusProxy.makeProxyWrapper(DBusSess
 const DisabledIcon = 'my-caffeine-off-symbolic';
 const EnabledIcon = 'my-caffeine-on-symbolic';
 const TimerMenuIcon = 'stopwatch-symbolic';
+const timerIcons = [
```

Comment:
```suggestion
const TimerIcons = [
```

If this is a global constant, should it use a capital instead?